packages/ui-components/stories/story-utils/api/queries/useMeasureData.js

```diff
@@ -0,0 +1,128 @@
+/*
```

Comment:
The `useData` hooks are designed to be the data integration between the api and the components. I have deliberately kept these outside of the components so that the components are as little aware of the api as possible.